api/src/services/DatastreamManager.ts

```diff
@@ -105,6 +105,51 @@ class DatastreamManager {
         await fedoraObject.modifyObjectLabel(title);
     }
 
+    async uploadProcessMetadata(pid: string, stream: string, metadata: Record<string, unknown>): Promise<void> {
+        const fedoraObject = FedoraObject.build(pid);
+        const tasks = ((metadata.tasks ?? []) as Array<Record<string, string>>)
+            .map((task) => {
+                return (
+                    `    <DIGIPROVMD:task ID="${task.id ?? 1}">\n` +
```

Comment:
You don't actually need these + if you wanna indent and create new lines. You just need to surround your paragraph in  this ``.